core/src/components/toggle/toggle.tsx

```diff
@@ -178,8 +184,29 @@ export class Toggle implements ComponentInterface {
     this.ionBlur.emit();
   };
 
+  private getSwitchLabelIcon = (mode: Mode, checked: boolean) => {
+    if (mode === 'md') {
+      return checked ? checkmarkOutline : removeOutline;
+    }
+    return checked ? removeOutline : ellipseOutline;
+  };
+
+  private renderOnOffSwitchLabels(mode: Mode, checked: boolean) {
+    const icon = this.getSwitchLabelIcon(mode, checked);
+
+    return (
+      <ion-icon
+        class={{
+          'toggle-switch-icon': true,
+          'toggle-switch-icon--checked': checked,
```

Comment:
Minor thing, but this convention does not appear to be used anywhere else in the framework. Can we make this decision as a team before adopting this syntax?